src/applications/claims-status/tests/e2e/08.claim-letters-keyboard-only.cypress.spec.js

```diff
@@ -0,0 +1,74 @@
+import claimLetters from './fixtures/mocks/claim-letters.json';
+import featureToggles from './fixtures/mocks/feature-toggles.json';
+
+describe('Claim Letters Page', () => {
+  context('feature toggle enabled', () => {
+    beforeEach(() => {
+      cy.intercept('GET', '/v0/claim_letters', claimLetters.data);
+      cy.intercept('GET', '/v0/feature_toggles?*', featureToggles);
+
+      cy.login();
+      cy.visit('track-claims/your-claim-letters');
+      cy.injectAxe();
+    });
+
+    it('Can tab to download link and pagination', () => {
+      cy.tabToElement('va-link').should('exist');
+
+      cy.tabToElement('va-pagination').should('exist');
+
+      cy.axeCheck();
+    });
+
+    it('Pagination buttons work properly', { includeShadowDom: true }, () => {
+      // 'Prev' button should not show on first page
+      cy.get('.pagination-prev li .button-prev').should('not.exist');
```

Comment:
I would use the `findByText` version of this instead. This selector is an implementation detail of the `va-paginate` component and if that component's html structure changes, this statement will fail. It looks like this is unavoidable with the `tabToElement` calls, but it's better to avoid coupling this test to the `va-paginate` component more than necessary